File: android/app/src/main/java/com/xirigo/ecommerce/feature/home/domain/model/DailyDeal.kt
package com.xirigo.ecommerce.feature.home.domain.model

data class DailyDeal(
    val productId: String,
    val title: String,
    val imageUrl: String?,
    val price: String,
    val originalPrice: String,
    val currencyCode: String,
    val endTime: Long,
)

File: ios/XiriGoEcommerce/Feature/Home/Domain/Model/DailyDeal.swift
import Foundation

// MARK: - DailyDeal

/// The daily deal section with countdown timer on the home screen.
struct DailyDeal: Equatable {
    let productId: String
    let title: String
    let imageUrl: String?
    let price: String
    let originalPrice: String
    let currencyCode: String
    let endTime: Date
}
